pkg/framework/host.go

```diff
@@ -63,9 +64,10 @@ Installation:
 type Host struct {
 	g8sClient *giantclientset.Clientset
 	k8sClient kubernetes.Interface
+	backoff   *backoff.ExponentialBackOff
 }
 
-func NewHost() (*Host, error) {
+func NewHost(b *backoff.ExponentialBackOff) (*Host, error) {
```

Comment:
We have a specific pattern in most places for these kind of things. Is there a reason why we should not apply it here? Like having a config structure. I do not see why this here should be different. The next guy has to add something and then the interface changes again and somebody has to do a lot of refactoring again just to get some other dependency in here only because we always only take the quick and dirty approach. The next thing we want to configure here is the helm/appr clients. So this is foreseeable already.

Reply:
Done PTAL.